internal/services/list_item/resource.go

```diff
@@ -97,6 +98,12 @@ func (r *ListItemResource) Create(ctx context.Context, req resource.CreateReques
 		return
 	}
 
+	err = pollBulkOperation(ctx, data.AccountID.ValueString(), createEnv.Result.OperationID.ValueString(), r.client)
```

Comment:
I don't know if we own the `ctx` instance, so I'm unclear what timeout exists already. Given that, should we have a new context `WithTimeout`?